packages/training/deploy/.env.example

```
@@ -0,0 +1,101 @@
+DOCKER_REGISTRY=your-username
+
+# Image tag for builds
+TAG=latest
+
+# =============================================================================
+# PLATFORM: RUNPOD
+# =============================================================================
+
+# RunPod API key for programmatic pod management
+# Get from: https://runpod.io/console/user/settings
+RUNPOD_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)